cpp/oneapi/dal/backend/primitives/reduction/reduction_dpc.cpp

```diff
@@ -88,6 +88,50 @@ sycl::event reduce_by_rows_impl(sycl::queue& q,
     return sycl::event{};
 }
 
+template <typename Float, typename BinaryOp, typename UnaryOp>
+sycl::event reduce_by_rows_impl(sycl::queue& q,
```

Comment:
Please write the description and include the dimensions of the input arrays there as well.